apps/minifront/src/components/dashboard/assets-table.tsx

```diff
@@ -41,7 +41,7 @@ export default function AssetsTable() {
                 </h2>
               </div>
 
-              <div className='max-w-72 truncate'>
+              <div className='max-w-72 text-muted-foreground'>
                 <AddressComponent address={a.address} />
               </div>
             </div>
```

Comment:
We appear to have lost the address part, before:

<img width="559" alt="Screenshot 2024-03-28 at 10 28 55 AM" src="https://github.com/penumbra-zone/web/assets/16624263/dfc5b454-ab7b-4c6f-8f92-721a3f73b567">

After:
<img width="579" alt="Screenshot 2024-03-28 at 10 28 42 AM" src="https://github.com/penumbra-zone/web/assets/16624263/7aeefe75-59cd-4899-a907-f020363511cd">